lib/solidus_subscriptions/configuration.rb

```diff
@@ -4,7 +4,7 @@ module SolidusSubscriptions
   class Configuration
     attr_accessor(
       :maximum_total_skips, :maximum_reprocessing_attempts, :churn_buster_account_id,
-      :churn_buster_api_key,
+      :churn_buster_api_key, :ignore_past_failed_installments_on_new_cycle,
```

Comment:
Can you also add this to the initializer template and explain what it does with a comment?

Reply:
Thanks, missed the initializer template!